docker-images/ubuntu/Dockerfile

```diff
@@ -61,11 +62,22 @@ ADD codebuild codebuild
 RUN set -eux; \
   export LD_LIBRARY_PATH=""; \
   . codebuild/bin/s2n_setup_env.sh; \
-  codebuild/bin/s2n_install_test_dependencies.sh; \
+  export PATH=$TEST_DEPS_DIR/clang/bin:$PATH; \
+  TESTS=integration codebuild/bin/s2n_install_test_dependencies.sh; \
```

Comment:
We need a pipeline to build these...

Reply:
Yeah that would be great to have. It would make sure the image stays up to date.